packages/core/src/connectors/index.ts

```diff
@@ -39,6 +39,17 @@ const loadConnectors = async () => {
         instance.metadata.logo = `data:image/svg+xml;base64,${data.toString('base64')}`;
       }
 
+      if (
+        packagePath &&
+        instance.metadata.logo_dark &&
+        !instance.metadata.logo_dark.startsWith('http') &&
```

Comment:
nit: better valid via `new URL(…)` (and it requires `try catch`)?

Reply:
We can do this, but what if the URL is valid, but the logo is not desired?
It seems that the validity check can not handle this case.

Reply:
maybe the schema is not HTTP but FTP or others?
maybe it is wrong to just check whether it starts with 'http'?
(you can keep it for now)

@logto-io/eng any good idea?

Reply:
Conclusion: keep it for now.